### DISCUSSION TITLE
convert bunge euler angles to hkl uvw

### DISCUSSION THREAD

**Discussion by rajesh-khatirkar** [mtex-toolbox/mtex#704] [Ask Anything] [answered]
Dear Ralf/Kilian

How to convert Bunge euler angles into hkl uvw? Is there any specific command for that.

regards
rajesh

**Comment by kilir** [comment, marked as answer]
Hi,
have a look at the `round2Miller` function.
Cheers, Rüdiger

**Reply by rajesh-khatirkar** [reply]
Thanks. 

I used round2Miller(orientation) to convert my EBSD data to hkl uvw. Now I want to get back my orientations for further processing.

I am using orientation('Miller',[h k l],[u v w],ss,cs) command to convert hkl uvw to orientation. It works fine when hkl uvw are
perfectly orthogonal.

I did some simulations and this orthogonality is slightly lost. I mean its not perfect. Is there a way round for this. My simulation gives me hkl uvw and since its deformation simulation, orthogonality is slightly lost.

Thanks in advance.

Rajesh
